### ISSUE TITLE
Overlay not on top of game Sea Of Thieves

### ISSUE THREAD

**Issue by ZicoXision** [baffler/Transparent-Twitch-Chat-Overlay#19]
Hey!
Great program, however it doesn't show OnTop in Sea Of Thieves :o

**Comment by baffler** [comment]
Check out the other issue https://github.com/baffler/Transparent-Twitch-Chat-Overlay/issues/5

It's mostly likely the type of fullscreen mode Sea of Thieves uses, I don't have the game so I can't test it myself. You typically have fullscreen exclusive mode (this doesn't allow any other windows on top) and windowed mode. There's a 3rd type that is pretty much the same as fullscreen exclusive but it behaves like windowed mode in some ways. I think some Directx 12 games have this behavior, but I know the games on the Windows Store is usually like this (tested it on State of Decay 2).

Basically that long explanation just means windows can't be rendered on top of the game no matter if it's fullscreen or windowed. I don't really have a fix for this right now, there may not really be one.

**Comment by ZicoXision** [comment]
Thank you for your reply.
I understand.
ITs a great program anyway, keep up the good work :D

**Comment by zepaladin** [comment]
Can confirm that the same happens with Forza Horizon 4. It's not exclusive FS but it's the kind of fullscreen window that auto-minimizes the moment you alt-tab or drag something over it.
